spec/models/document_type_selection/option_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "json"
```

Comment:
I don't think we need to `require "json"` does it not autoload correctly? I see it's already in spec/models/document_type_spec.rb and now spreading so be good to work out if it's essential.